tests/integration/src/test/scala/mesosphere/marathon/integration/TaskUnreachableIntegrationTest.scala

```diff
@@ -172,16 +174,121 @@ class TaskUnreachableIntegrationTest extends AkkaIntegrationTest with EmbeddedMa
 
       marathon.listDeploymentsForBaseGroup().value should have size 0
     }
+
+    "wipe pod instances with persistent volumes" in {
+
+      Given("a pod with persistent volumes")
+      val pod = residentPod("resident-pod-with-one-instance-wipe").copy(
+        instances = 1
+      )
+
+      When("The pod is created")
+      val createResult = marathon.createPodV2(pod)
+      createResult should be(Created)
+      waitForDeployment(createResult)
+      val taskId = marathon.status(pod.id).value.instances.head.containers.head.containerId.get
```

Comment:
this looks kinda ugly. and potentially useful for other tests. how about `def podTasks(id: PathId)` in `MarathonFacade`?